Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1839,7 +1844,11 @@
 #endif
 #if ENABLE(BLOCK_SET_EXCEPTION_PORTS)
             (with-filter (require-not (webcontent-process-launched))
+#if HAVE(HARDENED_MACH_EXCEPTIONS)
+                (allow mach-message-send (kernel-mig-routine task_register_hardened_exception_handler)))
```

Comment:
Ditto.

            (when (defined? 'task_register_hardened_exception_handler)
                (allow mach-message-send (kernel-mig-routine task_register_hardened_exception_handler))))

Reply:
can I do `defined?` and a `require-not`?